### ISSUE TITLE
How to prevent the "<a...></a>" from being inserted into changelog.md?

### ISSUE THREAD

**Issue by vajnorcan** [conventional-changelog/standard-version#246]
first of all, I'm using BitBucket, and there are some request to be able to configure the links in the change log (which would be great, as in BB they simply don't work) or remove any links whatsoever for the public usage.

The issue I'm having is that even when I use the correct message e..g "fix: config file fix" and run the standard-version commad, the resulting change log looks like this 
![image](https://user-images.githubusercontent.com/5698035/40632713-4083da14-632d-11e8-8595-b87594cb7096.png)

I'm using the SourceTree for committing the features/hotfixes (gitflow) but I'm not aware of any extra character that it might add.

So how can I prevent this anchor tag being added each time?

Thank you!

**Comment by loktionov129** [comment]
You can try to use [Lifecycle scripts](https://github.com/conventional-changelog/standard-version#lifecycle-scripts).

```
{
  "standard-version": {
    "scripts": {
      "postchangelog": "replace '<a name="\d.\d.\d"></a>' '' CHANGELOG.md"
    }
  }
}
```

But I'm not sure that it will work out.

**Comment by barmaglot92** [comment]
Hello! Please explain why you add this line <a name="1.1.2"></a>
What the reason ?

**Comment by loktionov129** [comment]
Just use https://github.com/semantic-release/semantic-release

**Comment by jhabdas** [comment]
> Hello! Please explain why you add this line <a name="1.1.2"></a>
> What the reason ?

@barmaglot92 `a` is an [anchor element](https://devdocs.io/html/element/a). they're a basic building block of the web. anchors allow users to use the _fragment identifier_ (`#`) in the URL to _deep link_ to a specific _anchor_ on the page.

here's an example of a deep link to a specific release in the changelog
https://github.com/comfusion/after-dark/blob/master/CHANGELOG.md#390-2018-05-03

> If i got correctly, you use it to find last release

you can use them to find any release with an anchor by updating the fragment identifier

**Comment by vajnorcan** [comment]
@barmaglot92 I haven't added it. It is being generated by the standard-version

@loktionov129 Using semantic-release is not really a solution atm, that's why I opted for the standard-version. Thanks

@jhabdas We know what anchor elements are :))

It turns out that it's the rendering of this file while previewing on bitbutcket.org ... it simply displays the anchor code though it shouldn't. Closing and I apologize for the confusion
